nextjs-app/src/pages/index.tsx

```diff
@@ -6,9 +6,10 @@ import { PieSpinner } from "@justeattakeaway/pie-spinner/dist/react"
 import { PieModal } from "@justeattakeaway/pie-modal/dist/react"
 import { PieCookieBanner } from "@justeattakeaway/pie-cookie-banner/dist/react"
 import { PieIconButton } from "@justeattakeaway/pie-icon-button/dist/react"
-import '@justeattakeaway/pie-icons-webc/IconClose';
+import { IconClose } from "@justeattakeaway/pie-icons-webc/dist/react/IconClose";
 import { PieSwitch } from "@justeattakeaway/pie-switch/dist/react"
 import { PieCard } from "@justeattakeaway/pie-card/dist/react"
+import Link from 'next/link'
```

Comment:
Semicolon police 🚓

Reply:
Let's add a ticket to set up linting in general!